examples/default/cluster/cluster.yaml

```diff
@@ -42,10 +42,10 @@ spec:
       cloud:
         controllerImage: "gcr.io/cloud-provider-vsphere/cpi/release/manager:v1.0.0"
       storage:
-        controllerImage: "vmware/vsphere-block-csi-driver:v1.0.0"
-        nodeDriverImage: "vmware/vsphere-block-csi-driver:v1.0.0"
+        controllerImage: "gcr.io/cloud-provider-vsphere/csi/release/driver:v1.0.1"
```

Comment:
This release has fix for https://github.com/kubernetes-sigs/vsphere-csi-driver/issues/70 right? We probably want to remove `hostNetwork: true` in our configs now? Can you do that here or in a separate PR?

Reply:
It does... I can add it to this PR. And it needs the `NODE_NAME` stuff if it isn't already there.

Reply:
yup, the liveness probes also use custom ports (because they conflicted with host network), we can remove those as well

Reply:
@andrewsykim Okay, I pulled in the changes from https://github.com/kubernetes-sigs/vsphere-csi-driver/pull/71/files.

Take a look if I did it right. :)

Reply:
oh, I still need to do those liveness probe ports like you said... Hopefully can do that in a bit.